lighthouse-core/test/chromium-web-tests/download-devtools.sh

```diff
@@ -14,6 +14,7 @@ then
   cd "$DEVTOOLS_PATH"
 
   # Update to keep current
+  git clean -fd
```

Comment:
```suggestion
  git reset --hard
  git clean -fd
```

tbh i think we want both.